bitbots_ros_control/config/controller.yaml

```diff
@@ -9,23 +9,4 @@ joint_state_controller:
 DynamixelController:
   type: bitbots_ros_control/DynamixelController
   joints:
-      - HeadPan
-      - HeadTilt
-      - LShoulderPitch
-      - LShoulderRoll
-      - LElbow
-      - RShoulderPitch
-      - RShoulderRoll
       - RElbow
-      - LHipYaw
-      - LHipRoll
-      - LHipPitch
-      - LKnee
-      - LAnklePitch
-      - LAnkleRoll
-      - RHipYaw
-      - RHipRoll
-      - RHipPitch
-      - RKnee
-      - RAnklePitch
-      - RAnkleRoll
```

Comment:
What's the purpose of the removed lines here?

Reply:
yes this was accidentially commited